include/bpf/libbpf.h

```diff
@@ -247,6 +247,8 @@ bpf_object__close(struct bpf_object* object);
  * @param[in] name The name to look for.
  *
  * @returns The map found, or NULL if none.
+ *
+ * @exception ENOENT The map was not found.
```

Comment:
These APIs are C apis and should never throw exceptions, so do not use `@exception`

Reply:
doxygen does not have a way to comment errno. @exception is an alternative.

Reply:
It looks like this is what the rest of the doxygen comments in this file uses. We should probably take a change to update the entire file to be consistent, though I am personally not sure exactly what tag we should use.

Reply:
Here is the tracking issue: https://github.com/microsoft/ebpf-for-windows/issues/3684